clientevent.go

```diff
@@ -62,3 +71,21 @@ func ToClientEvent(se Event, format EventFormat) ClientEvent {
 	}
 	return ce
 }
+
+// ToClientEvent converts a single server event to a client event.
+func HeaderedToClientEvent(se HeaderedEvent, format EventFormat) ClientEvent {
+	ce := ClientEvent{
+		Content:        RawJSON(se.Content()),
```

Comment:
Don't we need to handle redacted content?

Reply:
I don't think in this specific case, no, as if the event is redacted then it will have been updated in the database before now.

Reply:
Databases are dendrite stuff though, not GMSL stuff? How can we be sure?

Reply:
We can't be. It's up to Dendrite to do the right thing.